### PR TITLE
Feature/adhocbrev hvilenderett

### PR DESCRIPTION
**Pull request by hconhegg**

### CODE REVIEW

File: pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocInformasjonHvilendeRett4Aar.kt [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                text(
+                    Bokmal to "Du har rett til innsyn",
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "Du har rett til å se dokumentene i saken din. Se vedlegg «Dine rettigheter og mulighet for å klage» for informasjon om hvordan du går frem.",
+                )
+            }
+            title1 {
```
Comment (routsi) [inline pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocInformasjonHvilendeRett4Aar.kt:94]: Bruke frasen? (Felles.HarDuSpoersmaal)

File: pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocMidlertidigOpphoerHvilenderett10Aar.kt [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                    Bokmal to "Når etteroppgjøret for 2024 er klart må vi vurdere om du skal få et endelig vedtak om stans av uføretrygden din. Vi stanser uføretrygden din midlertidig allerede nå for å unngå feilutbetaling av uføretrygd, og for å være sikker på at det stemmer at du ikke har rett til uføretrygd i 2024.",
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "Dersom du ikke ønsker å beholde retten til uføretrygd, trenger du ikke foreta deg noe.",
+                )
+            }
+            title1 {
+                text(
```
Comment (routsi) [inline pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocMidlertidigOpphoerHvilenderett10Aar.kt:65]: Bruke frase? (Felles.RettTilAAKlage)

File: pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocMidlertidigOpphoerHvilenderett10Aar.kt [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "Du kan også kontakte oss per telefon for å få informasjon om hvordan du klager.",
+                )
+            }
+
+            title1 {
+                text(
```
Comment (routsi) [inline pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocMidlertidigOpphoerHvilenderett10Aar.kt:81]: Bruke frasen? (Felles.RettTilInnsyn)

File: pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocMidlertidigOpphoerHvilenderett10Aar.kt [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "Du har rett til å se dokumentene i saken din. Se vedlegg «Dine rettigheter og plikter» for informasjon om hvordan du går frem.",
+                )
+            }
+            title1 {
+                text(
+                    Bokmal to "Har du spørsmål",
```
Comment (routsi) [inline pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocMidlertidigOpphoerHvilenderett10Aar.kt:92]: Bruke frasen? (Felles.HarDuSpoersmaal)

File: pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocInformasjonHvilendeRett4Aar.kt [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+                        )
+                    }
+                    item {
+                        text(
+                            Bokmal to "Endringen vil gjelde fra kalendermåneden etter endringen av inntekt.",
+                        )
+                    }
+                }
+            }
+            title1 {
```
[the remark this answers is not in the document]
Comment (hconhegg) [reply pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocInformasjonHvilendeRett4Aar.kt]: Done

File: pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocInformasjonHvilendeRett4Aar.kt (hunk 1 above)
Comment (hconhegg) [reply pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/AdhocInformasjonHvilendeRett4Aar.kt]: Done

**Review by routsi — APPROVED** [review]

**Pull request merged**